examples/select_cells.md

```diff
@@ -12,39 +12,44 @@
 
 `client.select_cells(where='gene', ...)`:
 ```python
->>> cells_with_gene = client.select_cells(where='gene', has=['CASTOR2 > 1'], genomic_modality='rna')
+>>> gene_symbol = client.select_genes().get_list()[0]['gene_symbol']
+>>> cells_with_gene = client.select_cells(where='gene', has=[f'{gene_symbol} > 1'], genomic_modality='rna')
 >>> assert len(cells_with_gene) > 0
 
->>> cells_with_gene_details_with_values = cells_with_gene.get_list(values_included=['CASTOR2'])
->>> cells_with_gene_details_with_values[0]['values'].keys()
-dict_keys(['CASTOR2'])
+>>> cells_with_gene_details_with_values = cells_with_gene.get_list(values_included=[gene_symbol])
+>>> cells_keys = cells_with_gene_details_with_values[0]['values'].keys()
+>>> assert len(cells_keys) == 1 and gene_symbol in cells_keys
```

Comment:
```suggestion
>>> assert cells_keys == [gene_symbol]
```
maybe?

Reply:
Have to cast the keys as lists, but I will do that and make those changes.